From e9aa9d485e8408a2f113ece4eca54f202962fb89 Mon Sep 17 00:00:00 2001
From: rbuj <robert.buj@gmail.com>
Date: Fri, 22 Oct 2021 13:15:31 +0200
Subject: [PATCH 7/9] mouse: fix memory leak

---
 capplets/mouse/mate-mouse-properties.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/capplets/mouse/mate-mouse-properties.c b/capplets/mouse/mate-mouse-properties.c
index e057f7a8..7ca4cf1a 100644
--- a/capplets/mouse/mate-mouse-properties.c
+++ b/capplets/mouse/mate-mouse-properties.c
@@ -405,6 +405,7 @@ main (int argc, char **argv)
 	context = g_option_context_new (_("- MATE Mouse Preferences"));
 	g_option_context_add_main_entries (context, cap_options, GETTEXT_PACKAGE);
 	capplet_init (context, &argc, &argv);
+	g_option_context_free (context);
 
 	activate_settings_daemon ();
 
-- 
2.36.1

